src/protocol/util.c

```diff
@@ -47,6 +59,28 @@
 /**@{*/
 
 /**
+ * \def noise_init()
```

Comment:
\def can be left off because the doc is attached directly to the function definition.  Also, I usually prefer \fn for functions and \def for macros.